web/setup.py

```diff
@@ -593,5 +642,9 @@ def run_migration_for_others():
         run_migration_for_sqlite()
 
 
-if __name__ == "__main__":
+def main():
     app()
+
+
+if __name__ == "__main__":
```

Comment:
Why was this required?

Reply:
This is to fix running the `pgadmin4-cli` command in the case of the pip wheel package.

Ref: https://github.com/pgadmin-org/pgadmin4/issues/7116#issuecomment-1968685734